packages/crawler/src/report.ts

```diff
@@ -69,6 +69,13 @@ const lambdaHandler = async ({
     const name = `github/${gitHubRepositoryFullName}`;
     const revision = await git.revparse(["HEAD"]);
 
+    let report = require(`${jscpdReportLocalPath}/jscpd-report.json`);
```

Comment:
And, please use `const` instead of `let`. 
`const` guarantees it immutable.

Reply:
done